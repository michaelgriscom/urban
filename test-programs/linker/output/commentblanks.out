H:SCBLNK:0014:0014:0002:2012151,15,04,25:URBAN-LLM:0019:SCBLNK
T:000014:F4000000:SCBLNK
T:000015:F4000000:SCBLNK
E:0004:0002:SCBLNK
