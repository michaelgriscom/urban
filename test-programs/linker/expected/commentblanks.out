H:SCBLNK:0000:0014:0016:2012151,07,40,15:URBAN-LLM:0019:SCBLNK
T:000014:F4000000:SCBLNK
T:000015:F4000000:SCBLNK
E:0004:0002:SCBLNK
